recipes/speclite/build.sh

```diff
@@ -0,0 +1,29 @@
+# Speclite needs the astropy-helpers package for building, which however is not meant to be
```

Comment:
I'm not familiar with `astropy-helpers`, but this discussion seems relevant:

https://github.com/conda-forge/staged-recipes/pull/2719#issuecomment-291442855

Reply:
Thank you, that is relevant. However, speclite does not incorporate a version of astropy-helpers as other packages, so we need to get it from somewhere else. At the same time, as concluded in the conversation you linked, we really do not need to provide `astropy-helpers` as a package. My solution avoids that, but if you have better proposals I'm of course open to change.

Reply:
I just looked at the source distribution on pypi.io and it looks like there is an `astropy_helpers` directory.

Reply:
uhm... you're right, I understand now what is happening. Let me try locally with the command suggested in the conversation you linked:

`python setup.py install --offline --no-git --single-version-externally-managed --record record.txt`

Reply:
ok, yes, it works. Sorry I didn't get it the first time around. Now the recipe is a lot simpler and less hacky.